src/components/05-text-input/text-input.config.yml

```diff
@@ -0,0 +1,30 @@
+title: "Text input"
+status: "ready"
+collated: true
+context:
```

Comment:
Same goes for this config file. Once the template is updated to accept and `id` prop, we'll need to add them to all of the `context` objects in the config.